templates/repo/commits_table.tmpl

```diff
@@ -14,7 +14,7 @@
 						<input type="checkbox" name="all" id="all" value="true" {{.All}}>
 						<label for="all">{{.i18n.Tr "repo.commits.search_all"}} &nbsp;&nbsp;</label>
 					</div>
-					<button class="ui black tiny button" data-panel="#add-deploy-key-panel">{{.i18n.Tr "repo.commits.find"}}</button>
+					<button class="ui blue button" data-panel="#add-deploy-key-panel">{{.i18n.Tr "repo.commits.find"}}</button>
```

Comment:
```suggestion
					<button class="ui blue tiny button" data-panel="#add-deploy-key-panel">{{.i18n.Tr "repo.commits.find"}}</button>
```

Reply:
It looks like better without 'tiny' class i guess